_episodes/017-twir-341-342.md

```diff
@@ -0,0 +1,72 @@
+---
+title: "This Week in Rust - Issue 341 and 342"
+date: 2020-06-09T14:30:00Z
+file: https://audio.rustacean-station.org/file/rustacean-station/twir-2020-05-26.mp3
+duration: "11:21"
+length: "7780645"
+#reddit: (leave blank on initial publish, amend with link and uncomment this line after Reddit thread has been posted)
+---
+
+Nell Shamrell-Harrington — lead editor of This Week in Rust — takes you through highlights from this week's issue of TWiR.
+
+<!--
+The episode introduction goes here.
+The first paragraph should ideally be short, and is used in various
+places as a "short description" for the episode. Any subsequent
+paragraphs show up as "expanded description".
+-->
+
+### Contributing to Rustacean Station
+
+<!-- You can probably leave this as-is -->
+
+Rustacean Station is a community project; get in touch with us if you'd like to suggest an idea for an episode or offer your services as a host or audio editor!
+
+ - Twitter: [@rustaceanfm](https://twitter.com/rustaceanfm)
+ - Discord: [Rustacean Station](https://discord.gg/cHc3Gyc)
+ - Github: [@rustacean-station](https://github.com/rustacean-station/)
+ - Email: [hello@rustacean-station.org](mailto:hello@rustacean-station.org)
+
+### Referenced resources
+- [This Week in Rust 341](https://this-week-in-rust.org/blog/2020/06/02/this-week-in-rust-341/)
```

Comment:
Would it make sense to split this up by 341 and 342 using sub-headings like what we did in https://github.com/rustacean-station/rustacean-station.org/blob/master/_episodes/011-jake-yoshua-stjepan.md ?

Reply:
Good idea - doing that now :)

Reply:
Done!